lib/devos/importer.nix

```diff
@@ -0,0 +1,114 @@
+{ lib, ... }: let
+
+  mkProfileSet = prefix: name: { default = "${prefix}/${name}"; }
+  unpackProfileSet = profile: profile.default;
```

Comment:
To enforce contracts (in advance of `nickel`), we should tend to make more extensive use of those small contract enforcing wrappers.